.github/actions/setup-repo/action.yaml

```diff
@@ -12,7 +12,7 @@ runs:
     - name: Install stable
       uses: dtolnay/rust-toolchain@master
       with:
-        toolchain: 1.85.1
+        toolchain: 1.86.0
```

Comment:
can you leave a comment to show where people can find the latest supported version?

Reply:
can't we use the rust toolchain version automatically?